packages/react-dom/src/events/ChangeEventPlugin.js

```diff
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {enqueueEvents, processEventQueue} from 'events/EventPluginHub';
+import * as EventPluginHub from 'events/EventPluginHub';
```

Comment:
Why glob this instead of `import { runEventsInBatch } from 'events/EventPluginHub'`?